packages/block-editor/src/components/url-input/test/button.js

```diff
@@ -128,9 +133,13 @@ describe( 'URLInputButton', () => {
 
 	it( 'should close the form when user submits it', async () => {
 		const user = userEvent.setup();
-		const onChangeMock = jest.fn();
 
-		render( <URLInputButton onChange={ onChangeMock } /> );
+		function TestInputButton() {
+			const [ url, onChange ] = useState( '' );
+			return <URLInputButton url={ url } onChange={ onChange } />;
+		}
```

Comment:
I think it's worth adding a comment explaining why we're adding the wrapper component. It might be unclear to someone looking at the test without context.

Reply:
Comment added :+1: